docs/develop/create-addin-commands-unified-manifest.md

```diff
@@ -0,0 +1,601 @@
+---
+title: Create add-in commands with the unified Microsoft 365 manifest
+description: Configure the unified Microsoft 365 manifest to define add-in commands for Excel, Outlook, PowerPoint, and Word. Use add-in commands to create UI elements, add buttons or lists, and perform actions.
+ms.date: 04/07/2023
+ms.localizationpriority: medium
+---
+
+# Create add-in commands with the unified Microsoft 365 manifest
+
+Add-in commands provide an easy way to customize the default Office user interface (UI) with specified UI elements that perform actions. For an introduction to add-in commands, see [Add-in commands](../design/add-in-commands.md).
+
+This article describes how to configure the unified manifest to define add-in commands and how to create the code for [function commands](../design/add-in-commands.md#types-of-add-in-commands). 
+
+> [!TIP]
+> Instructions for creating add-in commands with the XML manifest are in [Create add-in commands with the XML manifest](create-addin-commands.md).
+
+> [!NOTE]
+> The unified manifest is currently supported only for Outlook Add-ins. It is in preview and shouldn't be used for production add-ins.
```

Comment:
Consider implying that the unified manifest will eventually be ready for production.
```suggestion
> The unified manifest is currently supported only for Outlook Add-ins. It's in preview and not yet supported for production add-ins.
```